src/features/user/TreeMapper/TreeMapper.module.scss

```diff
@@ -501,6 +509,7 @@
     border-top-left-radius: 0px;
     border-top-right-radius: 0px;
     box-shadow: none;
+    padding-top: 65px;
     > :first-child {
```

Comment:
The padding to the sides of the title is 50px on this screen, but it's 40 px elsewhere. Please correct this (check for different screen sizes)